site/zenodo_rdm/openaire/records/components.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Zenodo-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""OpenAIRE record component."""
+
+from flask import current_app
+from invenio_drafts_resources.services.records.components import ServiceComponent
+from invenio_records_resources.services.uow import TaskOp
+
+from zenodo_rdm.openaire.tasks import openaire_direct_index
+
+
+class OpenAIREComponent(ServiceComponent):
```

Comment:
This component should also have a `delete_record` step. However, that is implemented in rdm-records now:

# https://github.com/inveniosoftware/invenio-rdm-records/blob/ea5982e0ed72ffc915846019d773597796438dc1/invenio_rdm_records/services/components/record_deletion.py#L25

Reply:
Created an issue for this : https://github.com/zenodo/zenodo-rdm/issues/544